servicetalk-data-protobuf-jersey/src/main/java/io/servicetalk/data/protobuf/jersey/ProtobufMediaTypes.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright © 2022 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.data.protobuf.jersey;
+
+import io.servicetalk.concurrent.api.Publisher;
+import io.servicetalk.http.api.HttpHeaderValues;
+
+import javax.ws.rs.core.MediaType;
+
+/**
+ * Media type definitions for Protobuf.
+ */
+public final class ProtobufMediaTypes {
+    /**
+     * {@code "application/x-protobuf"}.
+     * @see #APPLICATION_PROTOBUF_TYPE
+     * @see HttpHeaderValues#APPLICATION_PROTOBUF
+     */
+    public static final String APPLICATION_PROTOBUF = "application/x-protobuf";
```

Comment:
I say we stick with `x-protobuf` which seems to be the most popular option and we can add more over time if necessary. It will require additional runtime checks for content type matching that we can avoid if not necessary.

Reply:
Consider renaming APPLICATION_PROTOBUF -> APPLICATION_X_PROTOBUF then in case we will need to add `application/protobuf` later